src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +90,13 @@ const Sidebar = () => {
         >
           {hasUnreadMessages && <div className="notification"></div>}
           <span>Messages</span>
+
+          {numberOfUnreadMessages > 0 && (
+            <span>
+              ({numberOfUnreadMessages}
+              {playAudio})
```

Comment:
As mentioned elsewhere in this review, this isn't required for the audio to play, as it will happen when the hook runs.

```suggestion
```